Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Language Features:
  * General: Support ``ContractName.functionName`` for ``abi.encodeCall``, in addition to external function pointers.
+ * Adds comparison operators for external function types.
```

Comment:
```suggestion
 * Adds equality-comparison operators for external function types.
```
If we want to be pedantic :-). ``<`` is also a comparison operator and it's not implemented for external function types here...

Reply:
Maybe also ``General:`` - but looking through the changelog we're not consistent about that anyways...

Reply:
Changed in the latest push.